app/views/shared/_veterans_crisis_line_modal.html.erb

```diff
@@ -38,9 +38,6 @@
               <p>Call TTY if you have hearing loss&nbsp;<a href="tel:+18007994889" aria-label="8 0 0. 7 9 9. 4 8 8 9." class="usa-link usa-link--external">800-799-4889</a></p>
             </li>
           </ul>
-          <p>
```

Comment:
I thought when Becca went over this she just meant to get rid of this icon on the phone numbers or am I misremembering?
<img width="20" alt="Screen Shot 2021-12-02 at 6 08 38 PM" src="https://user-images.githubusercontent.com/20211771/144523247-be2877b5-bd4d-4879-8f09-5bf5c0a45d31.png">

Reply:
Your correct... I've updated the code and PR.  Thx Elisa.. Yes - I'll wait til the category speech stuff is merged back in to Master... and get that merged in to this.. before merging this back to Master.